fv3config/fv3run/_native.py

```diff
@@ -50,11 +51,22 @@ def run_native(config_dict_or_location, outdir, runfile=None):
             _run_experiment(
                 localdir,
                 n_processes,
-                runfile_name=_get_basename_or_none(runfile),
+                python_args=_get_python_args(runfile),
```

Comment:
Might be worth using a more general name than "python args" in that you could actually just put the path to an `fv3.exe` for this arg if we support that at some point down the line, but also a more specific name in that when I read "python args" my gut reaction is "these are all python args, that's the language we're using. What function are they being passed to?".

Maybe "executable_args"? e.g. `_get_executable_args`?

Reply:
I changed to `python_command` and `_get_python_command`. `executable_args` sounds pretty vague to me.